base_layer/wallet/src/storage/sqlite_db/wallet.rs

```diff
@@ -631,24 +591,71 @@ impl WalletBackend for WalletSqliteDatabase {
 /// Master Public Key that is stored in the db
 fn check_db_encryption_status(
```

Comment:
Why cant we remove/change this function?
We are enforcing encryption, thus this should always return the XChaCha20Poly130
Its going to make this code much smaller and simplier

Reply:
Done